doc/sphinx/ui/mainviews/charview.rst

```diff
@@ -499,10 +499,10 @@ including the start and end points of the line itself. If there are more than
 two intersections then the x, y, and total distance between the first and last
 intersections are shown before the point list.
 
-[0] indicates the starting point of the measure line, and the (x-y) co-ordinates
```

Comment:
This is a formatting choice, not a typographical error.

Reply:
Reverted @1f5831e